packages/server/lib/cloud/api/studio/get_and_initialize_studio_manager.ts

```diff
@@ -47,6 +46,10 @@ const downloadStudioBundleToTempDirectory = async ({ studioUrl, projectId }: Opt
       encrypt: 'signed',
     })
 
+    if (!response.ok) {
+      throw new Error(`Failed to download studio bundle: ${response.statusText}`)
```

Comment:
What's the user experience with throwing this error?

Reply:
We catch it up stream and report it. I'll take a look at @astone123's PR to make sure it plays nicely there, but prior to this change it would actually have failed in a different (more confusing) way.